v1.0/backend/src/models/users.js

```diff
@@ -0,0 +1,79 @@
+const mongoose = require('mongoose')
+const bcrypt = require('bcryptjs')
+const jwt = require('jsonwebtoken')
+
+const userSchema = new mongoose.Schema({
+    name: {
+        type: String,
+        required: true,
+        trim: true
+    },
+    userId: {
+        type: String,
+        unique: true,
+        required: true,
+        trim: true,
+    },
+    schoolId: {
+        type: String,
+        required: true,
+        ref: 'School'
+    },
+    password: {
+        type: String,
+        required: true,
+        trim: true,
+        minLength: 7,
+        validate(value) {
+            if (value.toLowerCase().includes('password')) {
+                throw new Error('Password cannot contain "password"')
+            }
+        }
+    },
+}, {
+    timestamps: true
+})
+
+userSchema.index({userId: -1})
```

Comment:
@apurvabisht97 , any specific reason for index({userId: -1}) ? i.e. descending order